client/client.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/rpc"
+	"os"
+	. "rtclbedit/curp"
+	. "rtclbedit/shared"
+)
+
+func main() {
+	args := os.Args
+	if len(os.Args) != 3 {
+		fmt.Println("[usage]: " + args[0] + " <identifier> <configuration file>")
+		os.Exit(1)
+	}
+
+	name := args[1]
+	filename := args[2]
+	node_map := Parse(filename)
+	master_map, witness_map, _ := ParseByRole(node_map)
+	master := Map_begin(master_map)
```

Comment:
why do we have a map if it's just one thing? can we not just use a singular entry for this and pass by reference?

Reply:
Yeah it's lazy code-reusing, will change.